lib/couchdb/index.js

```diff
@@ -84,23 +85,56 @@ const upsert = R.curry((couch, bucket, keys, doc) =>
 )(doc)))
 
 
+const _findOneEqById = R.curry((couch, bucket, predicates) =>
```

Comment:
@jxbadam 
It's more like findOneBy, but findOneBy exists and it requires a prop and a value to be passed in. I need a function that take predicates, like how findWhereEq works.

This is a 'findOne' version of findWhereEq. The other difference is that it uses getById to speed things up, hence the name `_findOneEqById`. I'm open to better name for this if you can come up with one.